qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -18,6 +26,12 @@ You can download an example sample information file and prep information file fr
 Sample information file
 -----------------------
 
+Similar to a `QIIME metadata file, <http://qiime.org/documentation/file_formats.html#metadata-mapping-files>`__
```

Comment:
Agree and more cause the QIIME mapping file is basically the combination of both. Note that we can have in the FAQ the difference between QIIME and Qiita information files.